langserve/playground/src/components/CustomAnyOfRenderer.tsx

```diff
@@ -0,0 +1,34 @@
+import { JsonFormsDispatch, withJsonFormsAnyOfProps } from "@jsonforms/react";
+import {
+  rankWith,
+  createCombinatorRenderInfos,
+  JsonSchema,
+  isAnyOfControl,
+} from "@jsonforms/core";
+import { renderers, cells } from "../App";
+
+export const CustomAnyOfRenderer = withJsonFormsAnyOfProps((props) => {
+  const anyOfRenderInfos = createCombinatorRenderInfos(
+    (props.schema as JsonSchema).anyOf!,
+    props.rootSchema,
+    "anyOf",
+    props.uischema,
+    props.path,
+    props.uischemas
+  );
+
+  const selectedIndex = anyOfRenderInfos.length - 1;
```

Comment:
comment may not make sense -- i have no idea what the code is supposed to do

Why is this hard-coded to the last index?

Reply:
The idea here is to get rid of the tabs altogether, as it doesn't make sense to show all possible union options in a nice way and if I recall correctly, only non-prompt inputs do include `anyOf` in our templates. 

Can revisit later with a dropdown style input instead, but this should work fine for now.